agent/src/discovery_handler_manager/diagrams/dh_query.mmd

```diff
@@ -0,0 +1,18 @@
+sequenceDiagram
+    Configuration Controller -) DiscoveryHandlerRegistry: new_request()
+    alt a Handler exists for the Request
+        DiscoveryHandlerRegistry ->> DiscoveryHandlerRequest: Creates with filtered list of endpoints
```

Comment:
"endpoints" is a little overloaded here. Is this DH endpoints to device filtering that is passed to the DH as discovery properties?

Reply:
I looked at the `DHRequestImpl` structure which defines endpoints as discovered devices but i dont think that is what this diagram is referring to by "endpoints"

Reply:
Well, there is a bit of type passing here, the `DHRequestImpl` defines an endpoint as a receiver for discovered devices, that is the output stream from a discovery handler's endpoint connection.
So endpoint for `DHRequestImpl` is an open connection to a discovery handler, and in here to create that we pass the list of discovery handlers endpoints that have the right name (hence filtered list)

Reply:
Can we reword this, then, to clarify what filtered means. For example "Creates a discover connection with each DH for a Configuration"